### PR TITLE
797338

### PR DESCRIPTION
**Pull request by DaveC49**
changes in the import matcher to change U+R to U+C and R to C

### CODE REVIEW

**Comment by DaveC49** [comment]
This should be the correct commit rebased onto master.

**Comment by christopherlam** [comment]
@DaveC49 I have zealously fixed the PR. While creating a PR you'll need to set the target branch. See the 2nd button "base: maint" must be changed to "base: master" because the upcoming branch "master-jqplot-redirect" is based upon it.

image

**Comment by DaveC49** [comment]
Christopher -my bad still have the "L "plates on. Do you want me to  to create it again?

**Comment by christopherlam** [comment]
> Do you want me to to create it again?

No it's fine; fixed now. For next PR you'll just need to check the correct base. For now let's focus on spelling in https://github.com/Gnucash/gnucash/pull/554/commits/5e61b8d45e055ee5d9403cf0900395a86c39d19c#diff-750557fe0e2904625d6b2b1a8de5f2f9R755 :smiley:

File: gnucash/gtkbuilder/dialog-import.glade [hunk 1]
```diff
@@ -752,7 +752,7 @@
     <property name="text" translatable="yes">This transaction requires your intervention or it will NOT be imported.</property>
   </object>
   <object class="GtkTextBuffer" id="textbuffer5">
-    <property name="text" translatable="yes">Double click on the transaction to change the matching transaction to reconcile, or the destination account of the auto-balance split (if required).</property>
+    <property name="text" translatable="yes">Double click on the transaction to eitherchange the matching transaction in GnuCash, or the destination account of the auto-balance split (if required).</property>
```
Comment (jralls) [inline gnucash/gtkbuilder/dialog-import.glade]: s/eitherchange/either change/ and lose the comma after 'GnuCash'.

File: gnucash/gtkbuilder/dialog-import.glade [hunk 2]
```diff
@@ -740,7 +740,7 @@
     </action-widgets>
   </object>
   <object class="GtkTextBuffer" id="textbuffer1">
-    <property name="text" translatable="yes">Multiple transaction rows may be selected and a transfer account assigned to all selected rows. Use Ctrl Left Click or Shift-Click to select multiple rows and then Right Click to select a transfer account. Only rows with "A" checked can be added to a selection.</property>
+    <property name="text" translatable="yes">Multiple transaction rows may be selected and a transfer account assigned to all selected rows. Use Ctrl Left Click or Shift-Click to select multiple rows and then Right Click to select a transfer account. Only rows with "A" checked can be added to a selection. </property>
```
Comment (jralls) [inline gnucash/gtkbuilder/dialog-import.glade]: The only change on this line is that an unnecessary space got added at the end.

**Comment by gjanssens** [comment]
@jralls It looks like @DaveC49 implemented the requested changes. Do you want to re-review or approve before I merge ?

@DaveC49 can you squash the two commits together ? The command line tool `git rebase -i` can be used for that. If you find it too complicated, I'll do so when merging.

**Comment by jralls** [comment]
@gjanssens it's good, merge when ready.

**Comment by gjanssens** [comment]
Merged after squashing the fix into the main commit. Thanks!

**Pull request closed without merging**